### DISCUSSION TITLE
Access to flows after Trial period

### DISCUSSION THREAD

**Discussion by sanwat26** [postmanlabs/postman-flows#362] [General]
Hi Team,

I have just stumbled upon the Postman flows and it has helped reduce the test efforts by a huge margin, however it says your trail period ends in x days. My question is whether the flow i have would be accessible after trial period or not? lets say i am just exploring this and yet not fully convinced if i should be buying the monthly plans? Would i lose all the flows and subsequent efforts i put into creating them? or i can have access even then?

**Comment by shrey-bhagat** [comment]
Hey @sanwat26 ! Flows will be available for all users. You'll be able to access Flows even after the trial period.
